README.md

```diff
@@ -105,6 +105,32 @@ def bake(**goods):
         logger.error("No temperature defined!", extra=goods)
 ```
 
+#### Running Tests Manually
+Create your environment and install the test requirements
+```
+virtualenv venv 
+source venv/bin/activate
+python setup.py install 
```

Comment:
I believe `pip install .` or `pip install --editable .` is preferred.

http://stackoverflow.com/a/15731459/1547030